### DISCUSSION TITLE
Collab hour recording

### DISCUSSION THREAD

**Discussion by RVA2869** [govdirectory/website#328] [Ideas]
### Description

Quick question: why don't you guys record (and upload) the collab hour or stream it on different channels?
Like the not so weekly :) LIVE Wikidata editing. This could be beneficial for the project, the more people can see the project (and its still active) the more will join (hopefully).

**Comment by Ainali** [comment]
I think that would be something different. The collab hour isn't so much a show and tell, but a space to work collaboratively, and I am not sure if that would be very useful recorded. Often we can sit for 15 minutes just editing and only occasionally asking each other questions. I also think that it might be a barrier if people want to join and ask basic questions if they are recorded while doing so.
